css/style.css

```diff
@@ -5,15 +5,195 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+/*Header
+-----------------------------------------------------*/
+.header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 10px 50Px ;
+}
+
+.logoImg{
+    width: 20px;
+}
+
+.hamburgerMenu{
+    display: none;
+}
+
+
+.navList{
+    display: flex;
+    list-style: none;
+}
+
+.navItem{
+    padding: 15px;
+}
+
+.navLink{
+    text-decoration: none;
+    color: rgb(168, 162, 162); 
+}
+
+.makeBlack{
+    color:rgb(105, 97, 97);
+    font-weight:bold;
+} 
+
+.navLink:hover, .makeBlack:hover{
+    color: chocolate;
+}
+
+/*Introduction
+----------------------------------------------------*/
+
+.introduction{
+    background-image: url('../img/first-background.jpg');
+    background-size: 117%;
+    background-repeat: no-repeat;
+    height: 47vh;
+    color: white;
+    text-align: center;
+    padding:15% 0;
+   }
+
+   .introduction h1{
```

Comment:
Only the specification inside the `{}` should be indented. `.introduction h1{` itself should be at column 0.

Reply:
Done!